lib/openapi_client_generator.rb

```diff
@@ -253,13 +282,28 @@ def self.at(definition, parameterizer: OpenAPIClientGenerator::Endpoint::Positio
       end
     end
 
-    def self.resource_for_path(path)
+    def self.resource_for_path(path, operation_id)
       path_segments = path.split("/").reject{ |segment| segment == "" }
-      repo_resource = path_segments[3]
-      org_resource = path_segments[2]
+      operation_resource = operation_id.split("/")[0]
 
-      supported_resources = ["deployments","pages", "hooks", "releases", "labels", "milestones"]
-      resource = path_segments.first == "orgs" ? org_resource : repo_resource
+      if path_segments.first == operation_resource
```

Comment:
Considering changing how we're grouping the clients to be able to group them into more _subresource_ files that are shorter and easier to read, since `issues.rb` originally was generated to include `reactions.rb` as well. TODO: continue investigating how this generalizes to all paths

Reply:
Conclusion: Nah, I think this makes it more complicated than it's worth. The actual paths aren't set up to mirror the current break down of files and that's okay!